python-313/typing/readonly.py

```diff
@@ -0,0 +1,48 @@
+"""Demonstration of PEP 705: TypedDict: read-only items
+
+Use PyLance in VS Code by setting Python › Analysis: Type Checking Mode or run
+the Pyright CLI:
+
+    $ python -m pip install pyright $ pyright --pythonversion 3.13 .
```

Comment:
Should these two commands be on separate lines?

Reply:
Oops, yes, definitely. It seems that the automatic comment formatter I ran is a bit eager.